docs/wiki/Client.mediawiki

```diff
@@ -1,4 +1,63 @@
-==Client (dexc) ==
+=Guides=
+
+==Client Quick Start Installation==
+
+===Install===
+
+* It is recommended to have at least 2 GB of available system memory and 25 GB of free disk space to run the DEX client. For the most secure setup, you can also install in a fresh virtual machine.
+
+You can use the
+[https://github.com/decred/decred-release/releases '''dcrinstall'''] tool to
+install everything you need. '''dcrinstall''' will guide you through installation
+of all needed software and help set up your Decred wallet.
+
+<code>./dcrinstall --dcrdex</code>
+
+'''dcrinstall''' will create a directory for your decred binaries.
+For Linux and Mac, the directory is at ''~/decred''. For Windows, it's
+''%HOMEPATH%\decred''. Any commands listed below are assumed be run from this
+directory. Instructions are for Linux. For Windows, you will drop the
+<code>./</code> prefix for commands.
+
+'''WARNING''': If instead of using '''dcrinstall''' you decide to build from source and
+you plan to trade on mainnet, use the <code>release-v0.3</code> branch instead of <code>master</code>.
+Furthermore, if you build dcrd and dcrwallet from source, you '''must''' use their
+<code>release-v1.7</code> branches, not <code>master</code>.
```

Comment:
**release-v1.7.4** for both, right now.

Reply:
This is an interesting point.  The *branch* for all 1.7.x releases is `release-v1.7` while the latest *tag* will be something like `release-v1.7.4`.  I've been leaning toward release branches to avoid needing to update the docs for every release.

Reply:
Using release-v1.7...